Dockerfile

```diff
@@ -12,8 +12,9 @@ COPY requirements.txt /root
 RUN cd /root \
 && mkdir input/
 
-#COPY alerts/*.json /root/input/ 
-#RUN chmod -R a+rw /root/input/
+# Make sure to put alert files into the directory
+COPY alerts/*.json /root/input/
+RUN chmod -R a+rw /root/input/
 
 
 ##########################################
```

Comment:
These two commented commands below... I assume they were supposed to update and update the system. Should they remain commented for now?

Docker build logs mention at some point that a newer pip version is available.

Reply:
Let's uncomment it. Maybe takes longer to build but at least it will be updated

Reply:
Done